recipe/meta.yaml

```diff
@@ -42,4 +42,4 @@ about:
 extra:
   recipe-maintainers:
     - ocefpaf
-    - davidh-ssec
+    - djhoese
```

Comment:
@djhoese can you confirm if I got this right? You used to be @davidh-ssec, right?